rfid_tracker/app.py

```diff
@@ -155,6 +158,7 @@ def tracker(
                     session.add(new_action)
                     session.commit()
                     bye(user_in_db.first_name, user_in_db.last_name)
+                    slack.post(text=f"{user_in_db.first_name} just checked out. {n_occupied_spots+1} spots left.")
```

Comment:
We have multiple Dominiks for example, it would be nice to have both first name and last name :)

Reply:
Good call!